assets/sip_samples/newspaper_c44a0b0d-6e2f-4af2-9dab-3a9d447288d0/data/mets.xml

```diff
@@ -52,16 +52,14 @@
 
     <!-- structural map -->
     <structMap ID="uuid-98f8be79-d900-445f-9633-8813dd3ab616" TYPE="PHYSICAL" LABEL="CSIP">
-            <div ID="uuid-87aeca03-9761-4234-b668-7d6cec3081c5" LABEL="Metadata"
-            ADMID="uuid-06efacfd-cc03-4e8f-b98a-e17b9e7eee1e" 
-            DMDID="uuid-3936403d-133f-4765-b3b9-0a46df28db17 uuid-a4440db5-87f9-45af-819a-b966ca7f10fa" />
-            <div ID="uuid-60c6a853-b4fe-4658-ad8e-961adc15a00a" LABEL="Representations">
-                <div ID="uuid-5f92a639-0b45-4a9e-9c9e-e2a5a8764804" LABEL="representation_1">
-                    <mptr xlink:type="simple" xlink:href="./representations/representation_1/mets.xml" LOCTYPE="URL" />
-                </div>
-                <div ID="uuid-64055a8d-5f09-4cac-bb59-2726e3d624ff" LABEL="representation_2">
-                    <mptr xlink:type="simple" xlink:href="./representations/representation_2/mets.xml" LOCTYPE="URL" />
-                </div>
-            </div>
+        <div ID="uuid-87aeca03-9761-4234-b668-7d6cec3081c5" LABEL="Metadata"
+        ADMID="uuid-06efacfd-cc03-4e8f-b98a-e17b9e7eee1e" 
+        DMDID="uuid-3936403d-133f-4765-b3b9-0a46df28db17 uuid-a4440db5-87f9-45af-819a-b966ca7f10fa" />
+        <div ID="uuid-5f92a639-0b45-4a9e-9c9e-e2a5a8764804" LABEL="Representations/representation_1">
+            <mptr xlink:type="simple" xlink:href="./representations/representation_1/mets.xml" LOCTYPE="URL" xlink:title="representation_1" />
+        </div>
+        <div ID="uuid-64055a8d-5f09-4cac-bb59-2726e3d624ff" LABEL="Representations/representation_2">
+            <mptr xlink:type="simple" xlink:href="./representations/representation_2/mets.xml" LOCTYPE="URL" xlink:title="representation_2" />
```

Comment:
`xlink:title` moet de waarde van de `@ID` van de filegrp van de representatie in de filesec hebben